src/lamplib/src/genny/tasks/yaml_linter.py

```diff
@@ -26,37 +32,47 @@ def main(genny_repo_root: str):
         SLOG.error(
             "Found invalidly-named yaml files. Please correct and rerun ./run-genny lint-yaml."
         )
-        sys.exit(1)
+        has_errors = True
 
     all_have_descriptions = True
     for workload_yaml in workload_yamls:
-        if not check_description(workload_yaml):
+        if not check_required_fields(workload_yaml):
             all_have_descriptions = False
     if not all_have_descriptions:
         SLOG.error(
-            "The above YAML workloads lack a Description field. This field should be populated with a human-readable description "
-            "of the workload and its output metrics. After doing so, please re-run ./run-genny lint-yaml"
+            f"""
+            The above YAML workloads lack one or more of the required {", ".join(REQUIRED_FIELDS)} fields. 
+            These fields should be populated with human-readable values. After doing so, please re-run ./run-genny lint-yaml
+            """
         )
-        sys.exit(1)
+        has_errors = True
 
-    config_file_path = path.join(genny_repo_root, ".yamllint")
-    yamllint_argv = ["--strict", "--config-file", config_file_path] + all_yamls
+    if not no_format:
+        config_file_path = path.join(genny_repo_root, ".yamllint")
+        yamllint_argv = ["--strict", "--config-file", config_file_path] + all_yamls
 
-    SLOG.info(
-        "Linting workload YAML files with yamllint",
-        count=len(all_yamls),
-        yamllint_argv=yamllint_argv,
-    )
+        SLOG.info(
+            "Linting workload YAML files with yamllint",
+            count=len(all_yamls),
+            yamllint_argv=yamllint_argv,
+        )
 
-    yamllint.cli.run(yamllint_argv)
+        yamllint.cli.run(yamllint_argv)
+
+    if has_errors:
+        sys.exit(1)
 
 
-def check_description(yaml_path):
+def check_required_fields(yaml_path):
     workload = _load_yaml(yaml_path)
-    if "Description" not in workload:
-        SLOG.error(f"Genny workload {yaml_path} lacks a Description field.")
-        return False
-    return True
+    no_errors = True
+    for field in REQUIRED_FIELDS:
+        if field not in workload:
```

Comment:
NIT: Do you think it's also worth checking that the field actually has some text in it? There could be a loop hole here where users can just put the key in with an empty value.